src/main/app/public/locales/fi/translation.json

```diff
@@ -473,6 +473,10 @@
     "osaamisalalle-ei-loytynyt-kuvausta": "Osaamisalalle ei löytynyt kuvausta.",
     "alin-hyvaksytty-keskiarvo": "Alin hyväksytty keskiarvo",
     "painotukset": "Painotukset",
-    "erityiset-koulutustehtavat": "Erityiset koulutustehtävät"
+    "erityiset-koulutustehtavat": "Erityiset koulutustehtävät",
+    "lukiodiplomit": "Lukiodiplomit",
+    "yleiset-tavoitteet": "Yleiset tavoitteet",
+    "keskeiset-sisällöt": "Keskeiset sisällöt",
```

Comment:
Tässä avaimessa on käytetty ääkkösiä. En tiiä onko se ongelma, mut noissa muissa on pelkkiä aakkosia.

Reply:
Nämä ei tosiaan ole oikein yhdenmukaisia, mutta ääkköset sinänsä ei aiheuta ongelmia. En ainakaan tässä vaiheessa ala näitä yhdenmukaistaa, koska vaatisi myös käännösavainten muuttamisen lokalisointi-palvelussa.